subsys/nrf_security/src/core/Kconfig

```diff
@@ -19,4 +19,18 @@ config PSA_CORE_OBERON
 	select PSA_WANT_AES_KEY_SIZE_192
 	select PSA_WANT_AES_KEY_SIZE_256
 
+config PSA_CORE_LITE
+	bool "PSA core created for tiny footprint"
+	depends on SOC_SERIES_NRF54LX
+	help
+	  The PSA core with tiny footprint is created to be used e.g. for bootloader
+	  or similar use-cases where there is severe size-restrictions. This PSA core
+	  will only provide limited algorithm support and only a limited amount of features
+	  from the PSA crypto key management APIs. Note that volatile key-store is not
+	  possible to use.
+
 endchoice
+
+config PSA_CORE_LITE_NSIB_ED25519_OPTIMIZATIONS
+	bool "Optimize Ed25519 for NSIB usage"
```

Comment:
This can be confusing I think, to me that seems that it uses a PSA core lite where only the Ed25519 is supported. Not that we just optimise it.

Reply:
Adding helptext here

Reply:
Fixed